3 - PB/Documentazione esterna/Specifica Tecnica/Specifica Tecnica.typ

```diff
@@ -180,6 +180,19 @@ Di seguito sono elencati i requisiti minimi necessari per l'esecuzione dell'appl
 )
 
 #pagebreak()
+
+= Esecuzione del software
+
+Per avviare correttamente WMS3, è necessario seguire i seguenti passaggi:
+
+  - assicurarsi di avere Docker installato e funzionante sul proprio sistema;
+  - navigare fino alla directory contenente il file _compose.yml_;
+  - eseguire il comando "docker-compose up" da terminale per avviare i container pertinenti.
```

Comment:
il comando è `docker compose up -d` oppure anche `docker compose watch`. Entrambi in monospace